### ISSUE TITLE
Replicating DeT use limited dataset

### ISSUE THREAD

**Issue by 7yzx** [xiaozai/DeT#20]
Due to my hard drive capacity limitations, I only have access to 1200 GOT-10k depth sequences, as well as the full COCO depth dataset and DepthTrack. I'm trying to replicate DeT_DiMP50_Mean, but I find that the total loss remains around 1 and decreases very little. The validation set loss decreases, and the training speed is very slow, taking nearly an hour for each epoch. When I trained using only the COCO training set, the results were also not good; the training loss decreased, but the validation set loss stayed around 1, and the testing performance kept getting worse. I'm not sure what the problem is. Can I train a good DeT tracker with the limited dataset I have?

**Comment by 7yzx** [comment]
sorry , my mistake when testing, thank you for your excellent work in rgbd.
